### PR TITLE
Cloudwatch logging as a library

### PR DESCRIPTION
**Pull request by joselsegura**
# Description

Moving CloudWatch logging from Insights Results Aggregator to this project in order to be used by the different services

All the external data pipeline services written in Go should log into Cloudwatch, in order to get their logs on Kibana. As we already have one implementation in Insights Results Aggregator, I'm going to copy the source code here in order to be reused by the rest of the services

## Type of change
- New feature (non-breaking change which adds functionality)

## Testing steps

Regular CI

### CODE REVIEW

**Review by quarckster — APPROVED** [review]

File: logger/logger.go [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+// WorkaroundForRHIOPS729 keeps only those fields that are currently getting their way to Kibana
+// TODO: delete when RHIOPS-729 is fixed
+type WorkaroundForRHIOPS729 struct {
```
Comment (Sergey1011010) [inline logger/logger.go:40]: btw, maybe it was already fixed and we can delete a workaround

**Review by Sergey1011010 — APPROVED** [review]
Don't forget to update aggregator as well

**Comment by joselsegura** [comment]
> Don't forget to update aggregator as well

I already updated it, but I didn't push the change until I merge this one and create a new release for this repository. TYVM!

**Review by Bee-lee — APPROVED** [review]
LGTM, thanks.

**Comment by joselsegura** [comment]
After testing locally, I can say that it works in the same way as it works right now on aggregator side, so I'm merging it

**Pull request merged**